src/components/HomePage/InternAppBanner.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Typography, Link } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+	pinkText: {
+		color: theme.palette.secondary.main
+	},
+	purpleText: {
+		color: theme.palette.primary.main
+	},
+	descriptionWrapper: {
+		padding: theme.spacing(1)
+	},
+	wrapper: {
+		padding: theme.spacing(3),
+		textAlign: 'center',
+		display: 'flex',
+		justifyContent: 'center',
+		alignItems: 'center',
+		margin: theme.spacing(2, 'auto'),
+		width: 'fit-content',
+		border: 'solid 4px',
+		borderColor: theme.palette.secondary.main,
+		borderRadius: '16px',
+		backgroundColor: '#faddff5a',
+		[theme.breakpoints.down('xs')]: {
+			flexDirection: 'column'
+		}
+	}
+}));
+
+function InternAppBanner() {
+	const classes = useStyles();
+
+	return <div className={classes.wrapper}>
+		<div>
+			<Typography variant='h4' component='h2' align='center'>
+        Join our <span className={classes.pinkText}>team</span> ❤️
+			</Typography>
+			<Typography variant='h4' component='h2' align='center'>
+        We are <span className={classes.purpleText}>recruiting</span>!
+			</Typography>
+			<Typography align='center'>
+        Intern apps open until October 29, 2021 11:59pm PT
+			</Typography>
+		</div>
+		<div className={classes.descriptionWrapper}>
+			<Link component={PinkButton} href='https://forms.gle/nGEPFjSRzvNaciwC6'>
```

Comment:
I think we should open the link in a new tab, all our other button links do this too. 

Also, I didn't know we could do this. Did nesting a link inside a button not work?

Reply:
Oh wow I totally forgot I can use a button and make give it an 'a' component prop whoops.

Will fix.